docker/test/integration/features/http.feature

```diff
@@ -23,7 +22,6 @@ Feature: Sending data using InvokeHTTP to a receiver using ListenHTTP
 
   Scenario: A MiNiFi instance sends data through a HTTP proxy and another one listens
     Given a GetFile processor with the "Input Directory" property set to "/tmp/input"
-    And the "Keep Source File" property of the GetFile processor is set to "true"
```

Comment:
As far as I remember these were kept here to avoid flakiness. It could happen that the due to some environment issues the first HTTP request times out or fails in that case after the next GetFile invocation can still succeed and the test case succeeds. If this is removed the GetFile only runs once.